EstateAdministrationUI/Areas/Estate/Controllers/HomeController.cs

```diff
@@ -396,6 +476,16 @@ public IActionResult Index()
             return result;
         }
 
+        /// <summary>
+        /// Validates the model.
+        /// </summary>
+        /// <param name="viewModel">The view model.</param>
+        /// <returns></returns>
+        private Boolean ValidateModel(CreateOperatorViewModel viewModel)
```

Comment:
Codacy found an issue: [Remove this unused method parameter 'viewModel'.](https://app.codacy.com/manual/stuart_ferguson1/EstateAdministrationUI/pullRequest?prid=6044972)